templates/bot/csharp/command-and-response/GettingStarted.txt

```diff
@@ -15,10 +15,10 @@ Learn more
 -------------------------
 New to Teams app development or Teams Toolkit? Learn more about 
 Teams app manifests, deploying to the cloud, and more in the documentation 
-at https://aka.ms/teams-toolkit-vs-docs
+at https://aka.ms/teams-toolkit-vs-docs, https://aka.ms/teams-toolkit-advanced-tutorials
```

Comment:
Shall we provide more context on the second link?
As the context is "New to Teams app development or Teams Toolkit? Learn more about 
Teams app manifests, deploying to the cloud, and more in the documentation", it is not the context for the second link and will be confusing to the users (and even developers don't know opening 2nd link can solve their problems)